regimes/co/invoices.go

```diff
@@ -28,21 +28,28 @@ func (v *invoiceValidator) validate() error {
 				bill.InvoiceTypeCreditNote,
 				bill.InvoiceTypeProforma,
 			),
+			validation.Skip,
```

Comment:
It may not be obvious why this is necessary. For the benefit of future developers extending or looking at existing regimes to create new ones, would it be worth a comment explaining the need for the "skip"?

Reply:
True, I thought about this, but there are soooo many Skips! I think this is just something that needs to be confirmed by someone with experience in the PRs. Some regional guidelines/README would help.